tools/container/toolkit/toolkit.go

```diff
@@ -702,6 +716,7 @@ func generateCDISpec(opts *options, nvidiaCTKPath string) error {
 	cdilib, err := nvcdi.New(
 		nvcdi.WithMode(nvcdi.ModeManagement),
 		nvcdi.WithDriverRoot(opts.DriverRootCtrPath),
+		nvcdi.WithDevRoot(opts.DevRootCtrPath),
```

Comment:
Question -- do we also need to use `opts.DevRootCtrPath` when creating control device nodes in the code preceding this?

Reply:
Yes, we did. I have updated it.